extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -498,7 +548,12 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
    * @param sourceId the ID of the source
    */
   @SourceDsl
-  class Builder(val sourceId: String) {
+  class Builder(
+    val sourceId: String,
+    private val onGeoJsonParsed: (GeoJsonSource) -> Unit = {}
```

Comment:
should this be exposed to users?

Reply:
in case they want to get a callback, but by default we set it to {}

Reply:
Would like to hear from you about it 😄 
Don't have strong opinion yet, I guess using this builder directly in general may not be very common as we have dsl function

Reply:
it can be used with DSL as well, just looks a bit weird..

Reply:
Revisited.